flow-libs/cssnano.js.flow

```diff
@@ -0,0 +1,13 @@
+// @flow strict-local
+
+// Derived from the source of cssnano located at
+// https://github.com/cssnano/cssnano and its types at
+// https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cssnano/index.d.ts
+
+declare module 'cssnano' {
+  declare interface CSSNanoOptions {
+    configFile?: string;
+    preset?: [string, object] | string;
+  }
+  declare module.exports: ?CSSNano;
```

Comment:
Is  `CSSNano` defined anywhere? It probably silently became `any`

Reply:
I only see it being used in `CSSNanoOptimizer.js`

Reply:
I mean you're using the identifier `CSSNano` in this file but it's not declared anywhere 😄

Reply:
Oh I see, yea I think that should be `CSSNanoOptions` 😄

Reply:
It does export some kind of function: https://cssnano.co/docs/getting-started/#using-postcss-cli

Reply:
Hmm are you referring to `preset`? I added it in `CSSNanoOptions`

Reply:
You're currently exporting the object itself. But it's a function with the options as a parameter:
```
declare module.exports: (?CSSNanoOptions) => mixed;
```